src/components/Buttons/EditButton/EditButton.jsx

```diff
@@ -0,0 +1,17 @@
+import PropTypes from "prop-types";
+import { ICON_EDIT, ALT_TEXT_EDIT_ICON } from "src/common/constants";
+import "src/components/Buttons/buttons.scss";
+
+function EditButton({ onEdit }) {
+  return (
+    <button className="button-icon" onClick={onEdit}>
+      <img src={ICON_EDIT} alt={ALT_TEXT_EDIT_ICON} />
+    </button>
+  );
```

Comment:
Can we create a button component that can be utilized for all buttons in order to ensure reusability?

Reply:
Please do that, as components should be reusable.